layouts/partials/openapi/render-content-type.html

```diff
@@ -3,17 +3,11 @@
   Render a table showing content types and their descriptions, given
   two arrays with equal length:
 
-  * `content_types`: the content type strings
-  * `descriptions`: the description strings
-  * `added_in`: the "added in" version
-  * `changed_in`: the "changed in" dictionary
+  * `content_types`: OpenAPI data specifying the content types
```

Comment:
I believe the expected format should be more detailed, like saying its a dict with values being a dict with a `schema` key containing a JSON schema.

Reply:
Hm, I agree but this feels hard to describe in easy to understand words. Maybe we could link out to OpenAPI, e.g. https://swagger.io/docs/specification/media-types? Or alternatively, maybe a short form such as `{mime1: {schema: ...}, mime2: ...}` could work?

Reply:
I believe we usually try to have self-contained instructions (also your link has outdated information, we are using OpenAPI 3.1). Maybe a format that looks like something that would be rendered in the spec could work: `{string: {"schema": JsonSchema}}`

Reply:
That looks nice and succinct to me. Have made the change.